cmake/addons/CMakeLists.txt

```diff
@@ -254,11 +254,16 @@ foreach(addon ${addons})
     # try to find a perfect match
     list(FIND ADDONS_TO_BUILD ${id} idx)
     if(idx GREATER -1 OR "${ADDONS_TO_BUILD}" STREQUAL "all")
+      message(STATUS "Exact Match ${id}, building addon")
```

Comment:
minor: this msg is only correct if ADDONS_TO_BUILD != "all"

Reply:
I can just drop this change, was just useful to get a message to know if the exact match was hit.